src/scp/BallotProtocol.cpp

```diff
@@ -2119,12 +2107,12 @@ BallotProtocol::getJsonQuorumInfo(NodeID const& id, bool summary, bool fullKeys)
         delayed = n_delayed;
     }
 
-    auto f = LocalNode::findClosestVBlocking(*qSet, mLatestEnvelopes,
-                                             [&](SCPStatement const& st) {
-                                                 return areBallotsCompatible(
-                                                     getWorkingBallot(st), b);
-                                             },
-                                             &id);
+    auto f = LocalNode::findClosestVBlocking(
+        *qSet, mLatestEnvelopes,
+        [&](SCPStatement const& st) {
```

Comment:
not sure why this changed? Maybe you're using a different version of clang-format?

Reply:
I couldn't really get to replicate your setup of clang-format (maybe because I'm on Mac?).
Even with clang-format 5 I have diff with HEAD... Should I fix it according to what the CI is saying ?

Reply:
It's complaining in 3 places: https://travis-ci.org/stellar/stellar-core/jobs/658035681#L1257

Reply:
yeah the CI gives you the diff, so you can apply it and you should be good to go.
( in the future, you need to run `clang-format-5.0` not `clang-format` that is likely to pick up the wrong version )